versions.txt

```diff
@@ -55,6 +55,7 @@ google-cloud-redis:0.102.0-alpha:0.102.1-alpha-SNAPSHOT
 google-cloud-resourcemanager:0.102.0-alpha:0.102.1-alpha-SNAPSHOT
 google-cloud-scheduler:1.7.0:1.7.1-SNAPSHOT
 google-cloud-securitycenter:0.102.0:0.102.1-SNAPSHOT
+google-cloud-shared-config:0.0.0:0.0.1-SNAPSHOT
```

Comment:
we're releasing this?

Reply:
I *think* we have to release it. It's like how we release google-api-grpc and google-cloud-client's as poms to maven.

We could also wait until the split repo to actually do this. Doing it now will mean it gets versioned everytime we release the monorepo. When we have a split repo available to move it to, we would stop versioning it here.

Reply:
It does have to be released. Anything that is referenced as a parent is also loaded a use time.